django_mailbox/models.py

```diff
@@ -881,7 +881,8 @@ def __getitem__(self, name):
         return value
 
     def __str__(self):
-        return self.document.url
+        return f'{self.message}: {self.document.url if self.document else None}'
```

Comment:
Since self.message is nullable, this str is probably not deterministic enough: 
- 2 attachments without URLs on the same message will display the same str
What about using `get_filename` instead?
```suggestion
        if self.document:
            return f'{self.get_filename()}: {self.document.url}'
        return self.get_filename()
```